infra/modules/providers/azure/service-plan/main.tf

```diff
@@ -0,0 +1,16 @@
+resource "azurerm_resource_group" "svcplan" {
+  name     = "${var.resource_group_name}"
+  location = "${var.resource_group_location}"
+}
+
+resource "azurerm_app_service_plan" "svcplan" {
+  name                = "${var.svcplan_name}"
+  location            = "${azurerm_resource_group.svcplan.location}"
+  resource_group_name = "${azurerm_resource_group.svcplan.name}"
+  kind                = "${var.svcplan_kind}"
+
+  sku {
+    tier = "${var.svcplan_tier}"
+    size = "${var.svcplan_size}"
+  }
+}
```

Comment:
This module should also support tag configuration... Here's an [example ](https://github.com/Microsoft/bedrock/blob/42047255990d34b95a359943304f7b5cea066fa7/cluster/azure/backend-state/resource-group.tf#L4)in bedrock.